### PR TITLE
added references for Vinet

### PR DESCRIPTION
**Pull request by sannecottaar**
Minor changes to add references to Vinet EoS and remove rogue 'third-order's

### CODE REVIEW

**Comment by bobmyhill** [comment]
can you reset to 82fb19f (your first commit in this branch) and rebase -i master instead of merge? It's much neater than merging...

**Comment by sannecottaar** [comment]
Noted, merging was  the quick way github allowed a fix using their online editor.

File: burnman/eos/vinet.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# This file is part of BurnMan - a thermoelastic and thermodynamic toolkit for the Earth and Planetary Sciences
+  # This file is part of BurnMan - a thermoelastic and thermodynamic toolkit for the Earth and Planetary Sciences
```
Comment (bobmyhill) [inline burnman/eos/vinet.py]: Stray [  ] here...

File: burnman/eos/vinet.py [hunk 2]
```diff
@@ -11,10 +11,13 @@
...
-    Vinet equation of state.  Returns bulk
-    modulus in the same units as the reference bulk
-    modulus.  Pressure must be in :math:`[Pa]`.
+    compute the bulk modulus as per the
+    Vinet equation of state.  Reference bulk
+    modulus should be in :math:`[Pa]`.
+    References for this equation of state are :cite:`vinet1986`
+    and :cite:`vinet1987`
+    
+
```
Comment (bobmyhill) [inline burnman/eos/vinet.py]: Unnecessary new lines, period needed after citation.

Comment (bobmyhill) [inline burnman/eos/vinet.py]: References would be better in the blurb for the Base class (after what is currently line 57)

File: sphinx/ref.bib [hunk 3]
```diff
@@ -949,3 +949,25 @@ @article{kennett1991
...
+@article{vinet1986,
+title={A universal equation of state for solids},
+author={Vinet, PJJR and Ferrante, J and Smith, JR and Rose, JH},
+journal={Journal of Physics C: Solid State Physics},
+volume={19},
+number={20},
+pages={L467},
+year={1986},
+publisher={IOP Publishing}
+}
```
Comment (bobmyhill) [inline sphinx/ref.bib]: doi={10.1088/0022-3719/19/20/001}

File: sphinx/ref.bib [hunk 4]
```diff
@@ -949,3 +949,25 @@ @article{kennett1991
...
+@article{vinet1987,
+title={Temperature effects on the universal equation of state of solids},
+author={Vinet, Pascal and Smith, John R and Ferrante, John and Rose, James H},
+journal={Physical Review B},
+volume={35},
+number={4},
+pages={1945},
+year={1987},
+publisher={APS}
+}
```
Comment (bobmyhill) [inline sphinx/ref.bib]: doi={10.1103/PhysRevB.35.1945}

**Comment by sannecottaar** [comment]
Thanks Bob for your corrections.

**Pull request merged**